.github/workflows/ci.yml

```diff
@@ -6,20 +6,25 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  AWS_DEFAULT_REGION: us-east-1
+  AWS_BACKUP_REGION: eu-central-1
+  CLOUDFLARE_ACCOUNT_ID: 895762025d37fc687ecd72d7cc80204a
+  CLOUDFLARE_ZONE_ID: c192cf8ac042c681023493c52edd44c8
+
 jobs:
-  build:
+  setup:
     runs-on: ubuntu-latest-xl
-    env:
-      AWS_DEFAULT_REGION: us-east-1
-      AWS_BACKUP_REGION: eu-central-1
-      CLOUDFLARE_ACCOUNT_ID: 895762025d37fc687ecd72d7cc80204a
-      CLOUDFLARE_ZONE_ID: c192cf8ac042c681023493c52edd44c8
     steps:
       - name: Setup Deno environment
```

Comment:
The `setup-deno` step will need to be in every job that needs it rather than the top level.

Reply:
@wperron I've made these changes. From the looks of it, it looked like every job needed deno to be set up so I've added it as a step on each job